frontend/src/redux/store.ts

```diff
@@ -0,0 +1,9 @@
+import { createStore } from 'redux';
+import { rootReducer } from './root';
+
+const typed_window = window as any;

```

Comment:
More like `untyped_window`? :clown_face:

Reply:
I renamed it to `not_so_typed_window` to compromise